webapp/src/Controller/API/AbstractRestController.php

```diff
@@ -134,6 +134,12 @@ protected function renderData(
         if (!$request->query->has('strict') || !$request->query->getBoolean('strict')) {
             $groups[] = 'Nonstrict';
         }
+        if ($this->dj->checkrole('api_reader')) {
+            $groups[] = 'Restricted';
```

Comment:
Nice! I think we have more fields we only expose to api readers (submission files link at least) which we could also use this for. We now do (ugly) stuff with an event subscriber (see `SubmissionVisitor`)